mmv1/products/networkconnectivity/Spoke.yaml

```diff
@@ -131,6 +131,8 @@ properties:
       - linked_router_appliance_instances
       - linked_vpc_network
       - linked_producer_vpc_network
+    update_mask_fields:
+      - 'linkedVpnTunnels.includeImportRanges'
```

Comment:
Can you please add the update test for this field to this file https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/networkconnectivity/resource_network_connectivity_spoke_test.go?

This is the doc to add an update test https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test

Reply:
tests have been added